docs/assets/scss/_code_block.scss

```diff
@@ -12,7 +12,7 @@ main {
       line-height: 110%;
       max-width: 100%;
       background: #f2f6ff;
-      border: 1px solid #cbdbff;
+      border: 2px solid #f2f6ff;
```

Comment:
Not sure why you removed border? Doesn't work so well in tabs for example 
https://deploy-preview-17906--infallible-bardeen-164bc9.netlify.app/preview/explore/#set-up-yugabytedb-universe  and scroll down a little

Reply:
code blocks looks cleaner, but on tabs we have to do something else. I'll retain this for simple back ticks. I'll revert the change for code blocks.

Reply:
You need to bump the border width back to 1 as well

Reply:
![image](https://github.com/yugabyte/yugabyte-db/assets/79169168/b37366c6-8090-4bda-a91d-f4fa374b6516)

Not convinced...

Reply:
Ok, let's discuss this separately. When simple ticks are used in a sentence, it flows well. But the tables are striped. I'll revert it for now.